subsys/net/lib/dns/mdns_responder.c

```diff
@@ -423,7 +428,21 @@ static void send_sd_response(struct net_context *ctx,
 		service_type_enum = true;
 	}
 
-	DNS_SD_FOREACH(record) {
+	DNS_SD_COUNT(&rec_num);
+
+	while (rec_num > 0 || ext_rec_num > 0) {
```

Comment:
So this means that if there is different count of records, the minimum one wins and the rest of the other records are skipped. Just wanted to verify that this is the intention?